docs/events/api/timeline-api-get.asciidoc

```diff
@@ -0,0 +1,86 @@
+[[timeline-api-get]]
+=== Get Timelines or Timeline templates
+
+Get Timelines or Timeline templates.
+
+==== Request URL
+
+`GET <kibana host>:<port>/api/timelines`
+
+==== Request params
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`only_user_favorite` |Boolean a|Returns only favorite Timelines / Timeline templates if set to `true`. Default to `false`
+|No
+|`page_index` |Number |Page number of the result
+|No
+|`page_size` |Number |Items per page in result
+|No
+|`search` |String |The key word of the Timelines / Timeline templates you are searching for
+|No
+|`sort_field` |String |The order you want the result to be arranged. Can be `title`, `description`, `updated`, `created`
+|No
+|`sort_order` |String |`asc` or `desc`
+|No
+|`status` |String |Can be `active` or `immutable`. `active` returns all the custom Timelines / Timeline templates, whereas `immutable` returns Elastic prebuilt templates.
+|No
+|`timeline_type` |String |Use the `timeline` object's <<timeline-object-typeField, `timelineType`>> field
+to determine whether a timeline or a Timeline template is created, where:
+
+`default` returns Timelines, `template` returns Timeline templates
+|No
+
+|==============================================
+
+===== Example requests
+
+Get Timelines:
+
+[source,console]
+--------------------------------------------------
+GET api/timelines?page_size=10&page_index=1&sort_field=updated&sort_order=desc&timeline_type=default
+
+--------------------------------------------------
+
+
+=== Get Timeline / Timeline template by savedObjectId
+
+Get single Timeline or Timeline template by savedObjectId.
+
+==== Request URL
+
+`GET <kibana host>:<port>/api/timeline?id=<savedObjectId>`
+
+===== Example requests
+
+Get Timeline by savedObjectId:
+
+[source,console]
+--------------------------------------------------
+GET /api/timeline?id=<savedObjectId>
+
+--------------------------------------------------
+
+
+=== Get Timeline template by templateTimelineId
+
+Get single Timeline template by templateTimelineId.
+
+==== Request URL
+
+`GET <kibana host>:<port>/api/timeline?template_timeline_id=<templateTimelineId>`
+
+===== Example requests
+
+Get Timeline by savedObjectId:
```

Comment:
Consider something like `Get Timeline by templateTimelineId` if that makes sense here

Reply:
Yes, @andrew-goldstein you are right, thank you.